src/codegen/sys/cargo_toml.rs

```diff
@@ -92,19 +92,42 @@ fn fill_in(root: &mut Table, env: &Env) {
 
     {
         let features = upsert_table(root, "features");
-        let versions = env
+        let mut versions = env
             .namespaces
             .main()
             .versions
             .iter()
-            .filter(|&&v| v > env.config.min_cfg_version);
-        versions.fold(None::<Version>, |prev, &version| {
+            .filter(|&&v| v > env.config.min_cfg_version)
+            .collect::<Vec<_>>();
+        let max_version = versions.iter().max().map(|x| *x);
```

Comment:
Why not get last element in sorted array?

Reply:
Good point!